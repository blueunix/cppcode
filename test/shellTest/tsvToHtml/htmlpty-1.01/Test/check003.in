<body>
The <a name=5 href="http://info.cern.ch/hypertext/WWW/Addressing/Addressing.html">URL (Uniform Resource Locator) specification</A>.
</body>
