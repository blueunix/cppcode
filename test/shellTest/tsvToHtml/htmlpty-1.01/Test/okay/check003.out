<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<HTML>
    <HEAD>
        <TITLE>
            <!-- Please supply a descriptive title here -->
        </TITLE>
        <!-- Please supply a correct e-mail address here -->
        <LINK REV="made" HREF="mailto:beebe@plot79.math.utah.edu">
    </HEAD>
    <BODY>
        <P>
            The
            <A NAME=5
                HREF="http://info.cern.ch/hypertext/WWW/Addressing/Addressing.html">
                URL (Uniform Resource Locator) specification
            </A>.
        </P>
    </BODY>
</HTML>
